client/src/components/Recommendation.js

```diff
@@ -0,0 +1,53 @@
+import { useAuth0 } from "@auth0/auth0-react";
+import React, { useState, useEffect } from "react";
+import JSONPretty from 'react-json-pretty';
+
+
+//Recommendation component
+const Recommendations = () => {
```

Comment:
looks good!

Reply:
has the function passed testing?

Reply:
Thank you for the feedback Nayab, I added a error handler that returns error message if the method fails despite all the steps taken to ensure it does not.